reactDashboard/src/pages/Overview.jsx

```diff
@@ -72,10 +75,21 @@ const Overview = ({
     }
   }
 
+  useEffect(() => {
+    // Only attempt to fetch data once
+    if (healthBoardDataset != null || councilAreaDataset != null) {
```

Comment:
There's a bug here - if councilAreaDataset is loaded before healthBoardDataset, then getNhsCsvDataDateRange is called with null, councilAreaDataset. I think line 80 should be:

if (healthBoardDataset != null) {

This is a strong argument for adding some test coverage of the pages - even just checking they render.

Reply:
updated line 80